server/broadcast.go

```diff
@@ -220,6 +231,9 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) {
 			cond.Wait()
 		}
 		cond.L.Unlock()
+		if monitor.Enabled {
+			monitor.SegmentFullyTranscoded(nonce, seg.SeqNo, common.ProfilesNames(sess.Profiles), int(successes) == len(res.Segments))
```

Comment:
If the comparison is `len(segHashes) == len(res.Segments)` then that avoids the additional variable

Reply:
Fixed.